conflowgen/analyses/yard_capacity_analysis.py

```diff
@@ -20,7 +20,8 @@ class YardCapacityAnalysis(AbstractAnalysis):
     def get_used_yard_capacity_over_time(
             self,
             storage_requirement: typing.Union[str, typing.Collection, StorageRequirement] = "all",
-            smoothen_peaks: bool = True
+            smoothen_peaks: bool = True,
+            convert_to_teu: bool = True
```

Comment:
In this project, we usually prefer to provide both TEU and box numbers side-by-side. This has not yet been introduced to all previews and analyses but it is the desired intention. Instead of adding a parameter that switches between the unit box and TEU, we should rather return a named tuple. Examples can be found at https://github.com/1kastner/conflowgen/blob/main/conflowgen/descriptive_datatypes/__init__.py. Please follow that pattern.